packages/apollo-codegen-swift/src/codeGeneration.ts

```diff
@@ -168,10 +168,17 @@ export class SwiftAPIGenerator extends SwiftGenerator<CompilerContext> {
         throw new GraphQLError(`Unsupported operation type "${operationType}"`);
     }
 
+    const {
+      options: { namespace },
+      fragments
+    } = this.context;
+    const isRedundant = !!namespace;
+    const modifiers = isRedundant ? ["final"] : ["public", "final"];
+
     this.classDeclaration(
       {
         className,
-        modifiers: ["public", "final"],
+        modifiers: modifiers,
```

Comment:
Nit: since the variable name matches the key, the second `modifiers is redundant.
```suggestion
        modifiers,
```

Reply:
Thanks